src/test/java/com/solvd/meta/carina/CheckProductDetailScreenTest.java

```diff
@@ -0,0 +1,43 @@
+package com.solvd.meta.carina;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.qaprosoft.carina.core.foundation.utils.R;
+import com.solvd.meta.carina.gui.components.Header;
+import com.solvd.meta.carina.gui.components.Product;
+import com.solvd.meta.carina.gui.pages.DetailsProductPage;
+import com.solvd.meta.carina.gui.pages.InventoryPage;
+import com.solvd.meta.carina.gui.pages.LogInPage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.lang.invoke.MethodHandles;
+
+public class CheckProductDetailScreenTest implements IAbstractTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
you've defined LOGGER, but I don't see that you've used it. If it's unused you can remove it